storage/innobase/btr/btr0scrub.cc

```diff
@@ -49,54 +50,54 @@ uint srv_background_scrub_data_interval = (7 * 24 * 60 * 60);
 * statistics for scrubbing by background threads
 */
 static btr_scrub_stat_t scrub_stat;
-static ib_mutex_t scrub_stat_mutex;
-mysql_pfs_key_t scrub_stat_mutex_key;
+static ib_mutex_t       scrub_stat_mutex;
+mysql_pfs_key_t		scrub_stat_mutex_key;
 
 static uint scrub_compression_level = page_zip_level;
 
-/**************************************************************//**
+/**************************************************************/ /**
 Log a scrubbing failure */
-static
-void
+static void
 log_scrub_failure(
-/*===============*/
-	dict_index_t* index,     /*!< in: index */
-	btr_scrub_t* scrub_data, /*!< in: data to store statistics on */
-	buf_block_t* block,	 /*!< in: block */
-	dberr_t err)             /*!< in: error */
+    /*===============*/
```

Comment:
The old-style function header comments format poorly. Not sure if removing them is reasonable to ask in this PR (this would also entail moving /*!< in ... */ arg comments to function header comment

Reply:
I've updated the comments, will do the same for 8.0.

Reply:
"/*====*/" should go away too
```suggestion
```